src/css/netjsongraph.css

```
@@ -362,6 +362,45 @@
+}
+
+.njg-container .default-popup .njg-tooltip-key {
+  flex-basis: 45%;
+  font-weight: 600;
+  text-transform: capitalize;
+  color: black;
+}
+
+.njg-container .default-popup .njg-tooltip-value {
+  flex: 1;
+  word-break: break-word;
```

Stylelint (17.11.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)